src/components/TodoItem.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styles from './TodoItem.module.css';
+
+class TodoItem extends React.Component {
+  state = {
+    editing: false,
+  }
+
+  handleEditing = () => {
+    this.setState({
+      editing: true,
+    });
+  }
+
+  handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      this.setState({ editing: false });
+    }
+  }
+
+  render() {
+    const completedStyle = {
+      fontStyle: 'italic',
+      color: '#595959',
+      opacity: 0.4,
+      textDecoration: 'line-through',
+    };
+
+    // Destructuring todo
+    const { completed, id, title } = this.props.todo;
```

Comment:
Great use of de-structuring assignment! 
![](https://media.giphy.com/media/tIeCLkB8geYtW/giphy.gif)